### ISSUE TITLE
[Bug]: Warning message from Home Assistant on restart

### ISSUE THREAD

**Issue by Touch-of-Ink** [Lint-Free-Technology/uix#607]
### My Home Assistant Version

Core 2026.9.2
Frontend 20260826.7
Uix Version 8.2.0

### My UIX installation method

HACS

### My lovelace configuration method

GUI

### What I am doing

Restarting HA

### What I expect to happen

Not to get this warning: 

Logger: uix.8.2.0
Source: components/system_log/__init__.py:330
First occurred: 1:59:09 PM (1 occurrence)
Last logged: 1:59:09 PM

UIX (8.2.0): ha-state-icon already patched by 8.2.0! Make sure you are using the latest version of UIX. User: Suzi Browser: Mozilla/5.0 (Linux; Android 16; SM-S918U1 Build/BP4A.251205.006; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/152.0.7977.64 Mobile Safari/537.36 Home Assistant/2026.8.3-24000 (Android 16; SM-S918U1)

### What happened instead

I got this warning:
Logger: uix.8.2.0
Source: components/system_log/__init__.py:330
First occurred: 1:59:09 PM (1 occurrence)
Last logged: 1:59:09 PM

UIX (8.2.0): ha-state-icon already patched by 8.2.0! Make sure you are using the latest version of UIX. User: Suzi Browser: Mozilla/5.0 (Linux; Android 16; SM-S918U1 Build/BP4A.251205.006; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/152.0.7977.64 Mobile Safari/537.36 Home Assistant/2026.8.3-24000 (Android 16; SM-S918U1)

### Minimal steps to reproduce

1. Restart Home Assistant through Settings > Tools then Check Configuration Restart
2. check logs when HA has restarted
3.
...

### Include any yaml code here

```YAML

```

### Error messages from the browser console

```console

```

### By checking each box below I indicate that I ...

- [x] Understand that this is a channel for reporting bugs, not a support forum (https://community.home-assistant.io/).
- [x] Have made sure I am using the latest version of the integration.
- [x] Have searched Home Assistant logs for any relevant UIX errors or warnings.
- [x] Understand that failure to follow the template above may increase the time required to handle my bug-report, or cause it to be closed without further action.

**Comment by dcapslock** [comment]
Hi, if this happened on update then it can be expected until devices have reloaded. If this continues when you have not updated then please reopen and attached the UIX integration diagnostics which you can down load using the three button menu next to the single UIX entry in Devices and services.
